src/jit/lsra.h

```diff
@@ -260,15 +285,15 @@ class LocationInfoList final
             }
             prevListNode = listNode;
         }
-        assert(!"GetTreeNodeInfo didn't find the node");
+        assert(!"removeListNode didn't find the node");
         unreached();
     }
 
     //------------------------------------------------------------------------
     // GetTreeNodeInfo - retrieve the TreeNodeInfo for the given node
     //
     // Notes:
-    //     The TreeNodeInfoInit methods use this helper to retrieve the TreeNodeInfo for child nodes
+    //     The Build methods use this helper to retrieve the TreeNodeInfo for child nodes
```

Comment:
Above you used BuildNode, this probably should match.

Reply:
I'll change the other to "Build" as `BuildNode()` is the name of a specific method.